src/components/Chat/ChatContainer.js

```diff
@@ -49,7 +49,7 @@ class ChatContainer extends Component {
       status: "NotInitiated"
     };
 
-    this.submitChatInitiationHandler = this.submitChatInitiation.bind(this);
+    this.submitChatInitiationHandler = this.initiateChatSession.bind(this);
```

Comment:
What is this change for?

Reply:
This looks fine to me bec we added eventHandler to 
`EventBus.on("initChat", this.initiateChatSession.bind(this));`
So on cleanup we need to remove eventHandler 
```
 componentWillUnmount() {
    EventBus.off(this.submitChatInitiationHandler);
  }
```